packages/coordinator/src/services/communication/tracker/tracker-response-builder.ts

```diff
@@ -21,9 +21,12 @@ interface TrackerResponse {
 }
 
 const createTrackerError = (error: TrackerErrorString, details?: string | Array<string>): TrackerError => {
```

Comment:
Feel free to test this but I'm sure we can write the function like this which makes it a little more readable `const createTrackerError = (error: TrackerErrorString, details?: string | Array<string>): TrackerError => {
  let messageDetails : Array<string>

  if(Array.isArray(details)) {
    messageDetails = details
  }
    
  return {
    errorMessage: error,
    errorDetails: messageDetails ? messageDetails : []
  }
}`